app/models/referee_interface/reference_safeguarding_form.rb

```diff
@@ -0,0 +1,47 @@
+module RefereeInterface
+  class ReferenceSafeguardingForm
+    include ActiveModel::Model
+    attr_accessor :any_safeguarding_concerns, :safeguarding_concerns, :candidate
+
+    validate :presentce_of_any_safeguarding_concerns
+    validate :presentce_of_safeguarding_concerns
```

Comment:
```suggestion
    validate :presence_of_any_safeguarding_concerns
    validate :presence_of_safeguarding_concerns
```

Reply:
I think I had the same typo on the previous PR  :shame_emoji